packages/terra-form/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Unreleased
 ### Added
 * Add missing props-table
 
+### Breaking Changes
+
+* Select and SelectField no longer accepts a list of strings, but a list of objects to set their value/display
```

Comment:
Given that we plan to enhance this API yet again to offer more functionality that may be non passive. I'd vote for the time being keep the API passive and cause the choices prop to warn that it will be removed in a future release.